bing_bert/run_glue_classifier_bert_base.py

```diff
@@ -690,7 +691,15 @@ def main():
                         default=False,
                         action='store_true',
                         help="Whether to enable progressive layer dropping or not")
-
+    parser.add_argument(
+        '--preln',
+        action='store_true',
+        default=False,
+        help=
+        "Whether to display the breakdown of the wall-clock time for foraward, backward and step"

```

Comment:
The description appears to be wrong. Perhaps change it "Switching to the variant of Transformer blocks that use pre-LayerNorm."

Reply:
Of course, thanks Minjia :)